.github/workflows/tests.yml

```diff
@@ -192,13 +192,28 @@ jobs:
         run: |
           npm run test:web:integration:local_cluster
 
+  wake-up-cloud:
+    needs:
+      - node-unit-tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Wake up the cloud instance
```

Comment:
I'm wondering what problem we want to address? Idling is disabled on `GitHub actions` instance
![2024-08-23_10-32-04](https://github.com/user-attachments/assets/c382e2b1-b93f-4e3a-9bfe-aa1592d35acc)

Reply:
I see. Then, something else happens during the test runs that causes an occasional timeout.